### DISCUSSION TITLE
Module not found: Error: Can't resolve 'antd/es/version/token-meta.json'

### DISCUSSION THREAD

**Discussion by AleksandrDorofeev** [ant-design/ant-design#47337] [Q&A]
Can anyone tell me, I cloned the project, installed the dependencies, ran pnpm start in master branch and got an error? Am I doing something wrong?
`Module not found: Error: Can't resolve 'antd/es/version/token-meta.json'`

**Comment by jysim3** [comment]
I faced the same error. After digging for awhile, I find that my node version was not compatible. Using the following node version v18.20.3 (npm v10.7.0) fixed it
